prow/spyglass/lenses/junit/template.html

```diff
@@ -29,7 +29,7 @@
         <td colspan="2" style="padding: 0;">
           <table class="failed-layout">
             <tr class="failure-name">
-              <td class="mdl-data-table__cell--non-numeric test-name">{{$firstTest.Name}}&nbsp;<i class="icon-button material-icons arrow-icon">expand_more</i></td>
+              <td class="mdl-data-table__cell--non-numeric test-name">{{$firstTest.ClassName}}: {{$firstTest.Name}}&nbsp;<i class="icon-button material-icons arrow-icon">expand_more</i></td>
```

Comment:
This only changes test names for failed tests, is it intended?

Reply:
@EricEdens did you want all tests to show class or only failing in lens?

Reply:
I would expect the naming scheme to be consistent across tests regardless of their outcome.

Reply:
👍 will update across all outcomes